services/anki-sync-server/images/bin/entrypoint.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/sh
 # file: entrypoint.sh
 
+# allow entering shell if requested
```

Comment:
**question**: Do we want to allow access to the shell outside the CLI?

What is the value in this?

Reply:
That makes sense to me!